index.js

```diff
@@ -4,6 +4,10 @@ module.exports = app => {
   app.on(['pull_request.opened'], pullRequest)
 
   async function pullRequest(context) {
+    // Only allow PR's from our fork
+    if (!/repos\/erwinmombay/.test(context.payload.pull_request)) {
```

Comment:
Meeee? :)

Reply:
do we want to work off of just my fork, or do you want to also be able to test on your fork? doesnt hurt to add you 😄 so i will LOL